.changeset/odd-ducks-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+"tinacms": patch
+---
+
+Ensure `null` values aren't passed back from getStaticPropsForTina
```

Comment:
```suggestion
Ensure `undefined` values aren't passed back from getStaticPropsForTina
```
According to the error I got, "null" is fine, but undefined causes the problems:

> Error: Error serializing .variables returned from getStaticProps in "/projects".
Reason: undefined cannot be serialized as JSON. Please use null or omit this value.